extra-multimedia/sunshine/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=sunshine
+PKGDES="An open-source host-side implementation of NVIDIA's GameStream protocol"
+PKGSEC=video
+PKGDEP="pulseaudio libdrm libevdev icu wayland openssl ffmpeg"
```

Comment:
You can add `avahi` here to enable moonlight to detect the sunshine host automatically.

Reply:
Interesting, as it's a runtime optional dependency, I will put it into PKGRECOM instead.